packages/react-components/src/components/SendBox.tsx

```diff
@@ -188,11 +204,34 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
 
   const sendTextFieldRef = React.useRef<ITextField>(null);
 
+  /* @conditional-compile-remove(file-sharing) */
```

Comment:
This state can be added uncondtionally (but will stay `false` all the time in stable builds).

See below.